src/native/corehost/fxr/fx_muxer.cpp

```diff
@@ -572,6 +585,7 @@ int fx_muxer_t::execute(
             argv,
             new_argoff,
             mode,
+            false,
```

Comment:
nit: can you do `/*is_sdk_command*/` for all the ones where it is just passing false/true?